daemon/cmd/status.go

```diff
@@ -121,15 +121,20 @@ func (d *Daemon) getMasqueradingStatus() *models.Masquerading {
 		return s
 	}
 
+	localNode, err := d.nodeLocalStore.Get(context.TODO())
```

Comment:
If I understand the code correctly, `getMasqueradingStatus` is only called once during daemon startup. I assume it is guaranteed that the local node store has started by that point?